Taskfile.yml

```diff
@@ -39,13 +38,12 @@ tasks:
     vars:
       GORELEASER_VERSION: "0.62.5"
     cmds:
-      - npm install -g nodemon
       - go get -u github.com/golang/dep/cmd/dep
       - go get github.com/cespare/reflex || true
       - go get -u github.com/gobuffalo/packr/...
       - go get -u github.com/haya14busa/goverage
-      - '{{ if ne OS "windows" }} curl -L https://github.com/goreleaser/goreleaser/releases/download/v{{ .GORELEASER_VERSION }}/goreleaser_$(uname -s)_$(uname -m).tar.gz | tar -xz -C ${GOPATH}/bin{{ else }}ver>nul{{ end }}'
-      - '{{ if ne OS "windows" }} chmod +x ${GOPATH}/bin/goreleaser{{ else }}ver>nul{{ end }}'
+      - '{{ if ne OS "windows" }} curl -L https://github.com/goreleaser/goreleaser/releases/download/v{{ .GORELEASER_VERSION }}/goreleaser_$(uname -s)_$(uname -m).tar.gz | tar -xz -C ${GOPATH}/bin{{ else }}echo Skipped{{ end }}'
```

Comment:
what is the benefit of echoing only 'skipped' here with no context to understand what was skipped or why? That was the intention behind using ver>nul to just push the windows notification to the last step with a verbose comment explaining what is needed

Reply:
`ver>nul` not works for me, crashes with `Can't found ver in $PATH` (tested on Windows 10).

![2018-03-30-101513_1366x768_scrot](https://user-images.githubusercontent.com/1083576/38119198-4f379694-3403-11e8-8849-b7ea94cc6df9.png)

Reply:
Damn. Do you know of any working noop command we can use in windows then?

Reply:
I'll try smth like `echo Skipped>nul` a bit later. :)

Reply:
Huh, plain space solved the problem. :) I'll rebase commits and push again.